asciidoc-confluence-publisher-converter/src/main/java/org/sahli/asciidoc/confluence/publisher/converter/AsciidocConfluencePage.java

```diff
@@ -134,11 +135,14 @@ public static AsciidocConfluencePage newAsciidocConfluencePage(AsciidocPage asci
 
             Map<String, Object> userAttributesWithMaskedNullValues = maskNullWithEmptyString(userAttributes);
             Options options = options(templatesDir, asciidocPagePath.getParent(), pageAssetsFolder, userAttributesWithMaskedNullValues);
-            String pageContent = convertedContent(asciidocContent, options, asciidocPagePath, attachmentCollector, userAttributesWithMaskedNullValues, pageTitlePostProcessor, sourceEncoding, spaceKey);
 
-            String pageTitle = pageTitle(asciidocContent, userAttributesWithMaskedNullValues, pageTitlePostProcessor);
+            Document document = ASCIIDOCTOR.load(asciidocContent, options);
```

Comment:
looks like now we need to read document content from the FS only once to get title, tags and the body, so probably some small performance improvement here